### PR TITLE
feat(smart-contracts): cross-chain gov calls using zodiac multisig plugin

### LINKED ISSUE #12096: Script for cross chain gov
Script to gather all existing bridge calls (transferId) from governor proposal execution + track relayer fees + execute on destination timelock in Unlock Owner

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This introduces a simple proposal logic to pass DAO calls across the bridge using Connext module for Gnosis Zodiac plugin 

# Issues

Fixes #12096
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — APPROVED** [review]

File: packages/networks/src/networks/polygon.ts [hunk 1]
```diff
@@ -124,6 +128,12 @@ export const polygon: NetworkConfig = {
   },
   unlockAddress: '0xE8E5cd156f89F7bdB267EabD5C43Af3d5AF2A78f',
   url: 'https://polygon.technology/',
+  wrappedNativeCurrency: {
+    name: 'Wrapped MATIC',
+    symbol: 'WMATIC',
+    decimals: 18,
+    address: '0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270',
+  },
```
Comment (clemsos) [inline packages/networks/src/networks/polygon.ts:136]:
```suggestion
```

File: packages/networks/src/networks/polygon.ts [hunk 2]
```diff
@@ -4,6 +4,10 @@ export const polygon: NetworkConfig = {
   blockScan: {
     url: (address: string) => `https://blockscan.com/address/${address}`,
   },
+  bridge: {
```
Comment (julien51) [inline packages/networks/src/networks/polygon.ts:7]: Can we call this `governanceBridge` for more clarity?

File: packages/networks/src/networks/gnosis.ts [hunk 3]
```diff
@@ -4,6 +4,10 @@ export const gnosis: NetworkConfig = {
   blockScan: {
     url: (address: string) => `https://blockscan.com/address/${address}`,
   },
+  bridge: {
```
Comment (julien51) [inline packages/networks/src/networks/gnosis.ts:7]: same as above

**Review by julien51 — APPROVED** [review]
One minor name change. Can be done in a follow up PR

**Comment by clemsos** [comment]
> One minor name change. Can be done in a follow up PR

Yes I'll make the change in #12804 which hold the latest version of the types and networks package

**Pull request merged**